### PR TITLE
CORE-5084 Fix line endings for config files

### PR DESCRIPTION
**Pull request by davidcurrie**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.md text
+*.txt text
+*.yaml text
```
Comment (jacob-scott3141) [inline .gitattributes:11]: Do we always use .yaml or do we need to include .yml here?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+# Declare files that will always have LF line endings on checkout.
+*.sh text eol=lf
+*.conf text eol=lf
+*.properties text eol=lf
```
Comment (jacob-scott3141) [inline .gitattributes:16]: Will we also need to include .gradle files here?

**Review by jacob-scott3141 — COMMENTED** [review]
LGTM, only a couple small things I could see but I could be wrong.

File: .gitattributes (hunk 1 above)
Comment (davidcurrie) [reply .gitattributes:11]: It's always `.yaml` in this repo

File: .gitattributes (hunk 2 above)
Comment (davidcurrie) [reply .gitattributes:16]: Thankfully no Gradle in this repo!

**Comment by davidcurrie** [comment]
> LGTM, only a couple small things I could see but I could be wrong.

Key question is whether this fixes the problem for

**Pull request merged**